src/tag/examples.js

```diff
@@ -26,26 +27,40 @@ const colorIcons = {
 };
 
 //$FlowFixMe
-const tagStyles: Array<string> = Object.values(STYLE);
+const tagStyleKeys: Array<string> = Object.values(COLOR_STYLE_KEYS);
 
 export default {
   [tests.ALL_BASIC_COLORS]: () => {
     return (
       <React.Fragment>
         <div style={{width: '200px'}}>
-          {tagStyles.map(color => (
-            <Tag
-              key={color}
-              color={color}
-              onActionClick={(e, tag) => {
-                if (typeof tag === 'string') {
-                  // eslint-disable-next-line no-console
-                  console.log('Tag is clicked:' + tag);
-                }
-              }}
-            >
-              Color {color}
-            </Tag>
+          <Tag
+            key="default"
+            onActionClick={(e, tag) => {
+              if (typeof tag === 'string') {
+                // eslint-disable-next-line no-console
+                console.log('Tag is clicked:' + tag);
+              }
+            }}
+          >
+            Default Color
+          </Tag>
+          {tagStyleKeys.map(colorKey => (
+            <ThemeContext.Consumer key={colorKey}>
```

Comment:
We can remove the `ThemeContext.Consumer` usage in the examples now, correct?

Reply:
No- since we only have the color keys and not the values (the component needs the color hex values, not the color keys that are in the `tagStyleKeys`), we need a reference to the theme for the examples if we want to pull the values from the theme. Alternatively, I could just import the light theme directly for the demo, but that doesn't seem like something we want to do.

Reply:
We use this pattern in the [icon](https://github.com/uber-web/baseui/blob/master/src/icon/styled-components.js#L13) component.